### PR TITLE
feat(issue-details): Show static replay when error is not within the replay

### PR DESCRIPTION
**Pull request by malwilley**
This covers an edge case when the error event is not within the bounds of the replay. I don't know how prevalent this is, but it has happened at least once ([see here for an example](https://dev.getsentry.net:7999/issues/4705496067/events/1ca4960fcbf748e487829f05c470fb2e/)).

My solution is to fall back to the static replay preview component. I extracted the success state from the existing one into a new component so it can be used in the new clip component as well.

### CODE REVIEW

File: static/app/components/events/eventReplay/replayPreview.spec.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const render: typeof baseRender = children => {
         routes: router.routes,
       }}
     >
-      <OrganizationContext.Provider value={OrganizationFixture()}>
+      <OrganizationContext.Provider value={OrganizationFixture({slug: mockOrgSlug})}>
```
Comment (scttcper) [inline static/app/components/events/eventReplay/replayPreview.spec.tsx:91]: should be able to pass the organization to render? we shouldn't need `RouteContext.Provider` or `OrganizationContext.Provider` here

Comment (malwilley) [reply static/app/components/events/eventReplay/replayPreview.spec.tsx:91]: Ah yeah that's true, changed

File: static/app/components/events/eventReplay/staticReplayPreview.tsx [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  isFetching,
+  focusTab,
+  replayId,
+  fullReplayButtonProps,
+  replay,
+}: StaticReplayPreviewProps) {
+  const organization = useOrganization();
+  const routes = useRoutes();
+  const fullReplayUrl = {
+    pathname: normalizeUrl(`/organizations/${organization.slug}/replays/${replayId}/`),
```
Comment (scttcper) [inline static/app/components/events/eventReplay/staticReplayPreview.tsx:41]: normalizeUrl is already called in link components

File: static/app/components/events/eventReplay/replayPreview.spec.tsx [hunk 3]
```diff
@@ -88,11 +87,9 @@ const render: typeof baseRender = children => {
         routes: router.routes,
       }}
     >
-      <OrganizationContext.Provider value={OrganizationFixture()}>
-        {children}
-      </OrganizationContext.Provider>
+      {children}
     </RouteContext.Provider>,
```
Comment (scttcper) [inline static/app/components/events/eventReplay/replayPreview.spec.tsx:91]: you likely don't need the RouteContext either

**Review by scttcper — APPROVED** [review]

**Pull request merged**